src/audio/src/tune/sof_src_lite_int32.m

```diff
@@ -0,0 +1,25 @@
+%% Export full set of conversions for IPC4

```

Comment:
Minor, update the comment line to something like "Export conversions for src-lite" , also there's some ^M garbage in the end of line.